api/src/test/java/uk/gov/dhsc/htbhf/claimant/service/EligibilityAndEntitlementServiceTest.java

```diff
@@ -174,10 +179,12 @@ void shouldEvaluateClaimForGivenPaymentCycle() {
         verify(eligibilityAndEntitlementDecisionFactory).buildDecision(IDENTITY_AND_ELIGIBILITY_RESPONSE, VOUCHER_ENTITLEMENT, false);
     }
 
-    @Test
-    void shouldEvaluateClaimForGivenPaymentCycleWithEligibilityOverride() {
+    @ParameterizedTest
+    @MethodSource("childrenDobs")
```

Comment:
We expect no children to behave the same as having children looking at this test, just wondering why the need to parameterise this test?

Reply:
we pass the list of childrenDobs into a constructor _and an assertion_ - so each test does assert that different children's dobs are used. Without adding this the original test would have passed if the code had hard-coded an empty list. I guess we could simply use a non-empty list, but the parameterised test doesn't hurt.